### DISCUSSION TITLE
Does anyone have a way to use absolute import paths with nls?

### DISCUSSION THREAD

**Discussion by acx-yz** [tweag/nickel#1783] [Q&A]
The `-I` flag in `nickel export` was a really nice change for being able to store shared schemas/contracts in a module like directory. Unfortunately, using the absolute paths break with `nls` and afaict `nls` doesn't support the include flag.

Are people just using relative paths or is there some other way to make this sort of layout work?

```bash
# import "common/ports.ncl" instead of import "../../common/ports.ncl"
/deployments/whatever.ncl

/common => contracts for ports or something
```

**Comment by jneem** [comment]
I guess `-I` is a little more complicated, but I'd consider it a bug that nls doesn't honor the `NICKEL_IMPORT_PATH` env variable.

I'm a bit surprised that absolute paths don't work, though... if you wouldn't mind filing an issue, that would help me remember to look into both of these next week.

**Reply by acx-yz** [reply]
Thanks, opened an issue https://github.com/tweag/nickel/issues/1786
